5.5/s2i/bin/run

```diff
@@ -9,6 +9,7 @@ export TRACK_ERRORS=${TRACK_ERRORS:-OFF}
 export HTML_ERRORS=${HTML_ERRORS:-ON}
 export INCLUDE_PATH=${INCLUDE_PATH:-.:/opt/app-root/src:/opt/rh/php55/root/usr/share/pear}
 export SESSION_PATH=${SESSION_PATH:-/tmp/sessions}
+export SHORT_OPEN_TAG=${SHORT_OPEN_TAG:-OFF}
```

Comment:
we should keep the 5.5 behavior as it was, lest we break someone's app, so default this to On.